intel_extension_for_transformers/neural_chat/models/model_utils.py

```diff
@@ -840,12 +840,10 @@ def load_model(
 
                     if not use_tpp:
                         try:
-                            model = intel_ipex.optimize(
+                            model = intel_ipex.llm.optimize(
                                 model.eval(),
                                 dtype=torch_dtype,
                                 inplace=True,
-                                level="O1",
-                                auto_kernel_selection=True,
                             )
                         except AssertionError:
                             model = intel_ipex.optimize(
```

Comment:
Could you also change it? as I know, `ipex.llm.optimize` is recommanded to run bf16 LLMs inference by IPEX.